WinUIGallery/ControlPages/DesignGuidance/AccessibilityColorContrastPage.xaml

```diff
@@ -60,250 +63,241 @@
             TextWrapping="Wrap" />
-        <Grid
-            Padding="8"
-            Background="{ThemeResource CardStrokeColorDefaultBrush}"
-            ColumnSpacing="8"
-            CornerRadius="{StaticResource ControlCornerRadius}"
-            RowSpacing="8">
-            <Grid.RowDefinitions>
-                <RowDefinition Height="Auto" />
-                <RowDefinition Height="Auto" />
-                <RowDefinition Height="Auto" />
-            </Grid.RowDefinitions>
-            <Grid.ColumnDefinitions>
-                <ColumnDefinition Width="Auto" />
-                <ColumnDefinition Width="Auto" />
-                <ColumnDefinition Width="Auto" />
-                <ColumnDefinition Width="*" />
-            </Grid.ColumnDefinitions>
-
-            <TextBlock
-                Grid.Row="0"
-                Grid.Column="0"
-                Margin="12,0,0,0"
-                VerticalAlignment="Bottom"
-                Style="{ThemeResource BodyStrongTextBlockStyle}"
-                Text="Text color" />
+        <controls1:SampleThemeListener>
+            <Grid
+                Padding="8"
+                Background="{ThemeResource ControlExampleDisplayBrush}"
+                BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
+                BorderThickness="1"
+                CornerRadius="{StaticResource ControlCornerRadius}"
+                ColumnSpacing="8"
+                RowSpacing="8">
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto" />
+                    <RowDefinition Height="Auto" />
+                    <RowDefinition Height="Auto" />
+                </Grid.RowDefinitions>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="Auto" />
+                    <ColumnDefinition Width="Auto" />
+                    <ColumnDefinition Width="Auto" />
+                    <ColumnDefinition Width="*" />
+                </Grid.ColumnDefinitions>
 
-            <controls1:InlineColorPicker
+                <controls1:InlineColorPicker
                 x:Name="TextColorPicker"
-                Grid.Row="1"
-                Grid.Column="0"
-                Margin="12,0,0,12"
+                AutomationProperties.Name="TextColorPicker"
+                Grid.RowSpan="2"
+                Header="Text Color"
                 ColorChanged="TextColorPicker_ColorChanged"
                 Color="Black" />
 
-            <TextBlock
-                Grid.Row="0"
-                Grid.Column="1"
-                Margin="12,10,0,0"
-                Style="{ThemeResource BodyStrongTextBlockStyle}"
-                Text="Background color" />
-            <controls1:InlineColorPicker
+                <controls1:InlineColorPicker
                 x:Name="BackgroundColorPicker"
-                Grid.Row="1"
+                AutomationProperties.Name="BackgroundColorPicker"
```

Comment:
Should the UIA name be a bit more descriptive here and on the other picker?

Reply:
I thought it was descriptive haha. All ears for ideas.

Reply:
I thought the UIA name should be something that you just have as the text to replace the button with, i.e. it also is semantically fitting. So in this case maybe something like "Pick Background color", given that the elements inside it already have the appropriate names, do we even need this?

Reply:
Fair point, I think this is good as is.